pages/linux/dmesg.md

```diff
@@ -4,20 +4,32 @@
 
 - Show kernel messages:
 
-`dmesg`
+    `dmesg`
 
 - Show kernel error messages:
 
-`dmesg --level err`
+    `dmesg --level err`
 
 - Show kernel messages and keep reading new ones, similar to `tail -f` (available in kernels 3.5.0 and newer):
 
-`dmesg -w`
+    `dmesg -w`
 
 - Show how much physical memory is available on this system:
 
-`dmesg | grep -i memory`
+    `dmesg | grep -i memory`
 
 - Show kernel messages 1 page at a time:
 
-`dmesg | less`
+    `dmesg | less`
+
+- Show kernel messages with a timestamp (available in kernels 3.5.0 and newer):
```

Comment:
I'm unsure that we need to include `(available in kernels 3.5.0 and newer)` here. Are there any current Linux distributions that use a kernel older than this in their latest release?

Reply:
kernels like  2.6.32-642.4.2.el6.x86_64  dont have these options, centos 6 for example which is still fairly common

Reply:
....I see. Checking the Centos website reveals that there's still a year or two of official support left there, so this should be ok then :-)

It just surprised me - I haven't run into a 2.6.x kernel for _ages_! (The last one was the OVH VPS kernel - ewwwwwww........)